brainiak/utils/fmrisim.py

```diff
@@ -443,11 +447,9 @@ def generate_stimfunction(onsets,
     if len(weights) == 1:
         weights = weights * len(onsets)
 
-    # How many elements per second are you modeling
-    temporal_resolution = 1000
-
-    # Generate the time course as empty, each element is a millisecond
-    stimfunction = [0] * round(total_time * temporal_resolution)
+    # Generate the time course as empty, each element is a millisecond by
+    # default
+    stimfunction = [0] * round(int(total_time * temporal_resolution))
```

Comment:
You bumped into a NumPy issue:
https://github.com/numpy/numpy/issues/5700

I suggest you create an `int` using the result of `round`, which preserves the rounding instead of truncating:
```int(round(total_time * temporal_resolution))```